uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/util/commands/CreateRepository.java

```diff
@@ -79,6 +112,13 @@ public CreateRepository(final File repoDir,
 
             final org.eclipse.jgit.api.Git git = new org.eclipse.jgit.api.Git(repo);
 
+            if(!sslVerify) {
+                StoredConfig config = repo.getConfig();
+                config.setBoolean( "http", null, "sslVerify", false );
+                config.save();
+            }
```

Comment:
Imho we should do something when `sslVerify` is true and repositories have the property already set to `false`.